.github/workflows/ci.yml

```
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set up Ruby
-        uses: ruby/setup-ruby@v1.68.0
+        uses: ruby/setup-ruby@v1.70.0
```

Go major version to always have latest
```suggestion
        uses: ruby/setup-ruby@v1
```